Gemfile

```diff
@@ -24,7 +24,7 @@ gem 'dough-ruby',
 gem 'jquery-rails'
 gem 'kaminari'
 gem 'letter_opener', group: :development
-gem 'mas-rad_core', '0.0.75'
+gem 'mas-rad_core', '0.0.76'
```

Comment:
I'm assuming the bump is to get access to new rad_core functionality required for this to work?